docs/source/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changes in nbconvert
 ====================
 
+4.2 (in development)
+--------------------
+
+``4.2 on GitHub - Add link once release is tagged``
```

Comment:
The milestone link always exists: https://github.com/jupyter/nbconvert/milestones/4.2